### ISSUE TITLE
Fixed: An error occurred while retrieving information for the 'ELLALoader' node.

### ISSUE THREAD

**Issue by Aenon1** [TencentQQGYLab/ComfyUI-ELLA#11]
I updated ComfyUI and all extensions before proceeding with this install. I ran the update through manager and also did a git pull.

I tried installing this through manager in ComfyUI, I tried to 'Update' through manager, and, I also tried using git clone after doing an 'uninstall' through manager and deleting the directory, all result in this not installing correctly and ella erroring out on load.
I installed the requirements through ComfyUI's python.
I downloaded everything as shown in the instructions. The file names and directory structure are the same as shown in your install instructions.

This is what I receive when I open ComfyUI:

[ERROR] An error occurred while retrieving information for the 'ELLALoader' node.
Traceback (most recent call last):
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\server.py", line 414, in get_object_info
    out[x] = node_info(x)
             ^^^^^^^^^^^^
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\server.py", line 392, in node_info
    info['input'] = obj_class.INPUT_TYPES()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-ELLA\ella.py", line 156, in INPUT_TYPES
    "name": (folder_paths.get_filename_list("ella"),),
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\folder_paths.py", line 225, in get_filename_list
    out = get_filename_list_(folder_name)
          ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\folder_paths.py", line 192, in get_filename_list_
    folders = folder_names_and_paths[folder_name]
              ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'ella'

[ERROR] An error occurred while retrieving information for the 'T5TextEncoderLoader #ELLA' node.
Traceback (most recent call last):
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\server.py", line 414, in get_object_info
    out[x] = node_info(x)
             ^^^^^^^^^^^^
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\server.py", line 392, in node_info
    info['input'] = obj_class.INPUT_TYPES()
                    ^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\custom_nodes\ComfyUI-ELLA\ella.py", line 176, in INPUT_TYPES
    for search_path in folder_paths.get_folder_paths("ella_encoder"):
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "D:\Ai\ComfyUI\ComfyUI_windows_portable\ComfyUI\folder_paths.py", line 134, in get_folder_paths
    return folder_names_and_paths[folder_name][0][:]
           ~~~~~~~~~~~~~~~~~~~~~~^^^^^^^^^^^^^
KeyError: 'ella_encoder'

**Comment by JettHu** [comment]
fixed by [PR #8](https://github.com/TencentQQGYLab/ComfyUI-ELLA/pull/8)

**Comment by Aenon1** [comment]
Thanks, title changed.
